lib/entity/domain-contact.php

```diff
@@ -47,6 +47,13 @@ class Domain_Contact {
 	 */
 	private Contact_Disclosure $contact_disclosure;
 
+	/**
+	 * Constructs a `Domain_Contact` entity
+	 *
+	 * @param \Automattic\Domain_Services\Entity\Contact_Id|null $contact_id
+	 * @param \Automattic\Domain_Services\Entity\Contact_Information|null $contact_info
+	 * @param \Automattic\Domain_Services\Entity\Contact_Disclosure|null $disclose_fields
```

Comment:
I think we can remove `\Automattic\Domain_Services\Entity\` from these since that's the namespace we are in.